packages/go_router/README.md

```diff
@@ -1,13 +1,14 @@
-# Welcome to go_router!
+# A Declarative Routing Package for Flutter
```

Comment:
nit: most of the other projects in flutter/packages put either the package name or title-cased name of the package in the headline

```suggestion
# go_router

A declarative routing package for Flutter.
```